src/bijectors/stacked.jl

```diff
@@ -37,74 +37,65 @@ struct Stacked{Bs, N} <: Bijector{1} where N
         ranges::NTuple{N, UnitRange{Int}}
     ) where {N, A<:AbstractArray{<:Bijector}}
         @assert length(bs) == N "number of bijectors is not same as number of ranges"
-        @assert all(isa.(bs, ZeroOrOneDimBijector))
+        @assert all(b -> isa(b, ZeroOrOneDimBijector), bs)
         return new{A, N}(bs, ranges)
     end
 end
 Stacked(bs, ranges::AbstractArray) = Stacked(bs, tuple(ranges...))
 Stacked(bs) = Stacked(bs, tuple([i:i for i = 1:length(bs)]...))
 
-isclosedform(b::Stacked) = all(isclosedform.(b.bs))
+isclosedform(b::Stacked) = all(isclosedform, b.bs)
 
 stack(bs::Bijector{0}...) = Stacked(bs)
 
 # For some reason `inv.(sb.bs)` was unstable... This works though.
 inv(sb::Stacked) = Stacked(map(inv, sb.bs), sb.ranges)
-@generated function inv(sb::Stacked{A}) where {A <: Tuple}
```

Comment:
The reason why this is here, rather than just using the `map` impl from above is that for large `Stacked`, i.e. large number of stacked bijectors, the result of `map` isn't type-inferrable. Thus, I don't think we should remove this

Reply:
Ok I will bring it back with a comment. Thanks.